tests/test_deprecated.py

```diff
@@ -34,6 +34,7 @@ def test_tbd_remove_in_v0_8_0_trainer():
         'max_nb_epochs': 'max_epochs',
         'min_nb_epochs': 'min_epochs',
         'nb_sanity_val_steps': 'num_sanity_val_steps',
+        'fast_dev_run': 'unit_test',
```

Comment:
This makes the test fail because in the trainer you set the max_epochs = 1 if self.unit_test, then the deprecated arg max_nb_epochs will have a different value (3) than max_epochs (1)

Reply:
Thanks a lot for helping. Shall i override the values in the test with the expected values i.e explicitly set values
`#### Override unit_test attributes for max_epochs and num_sanity_val_steps`
`kwargs['max_nb_epochs']=1 `
`kwargs['nb_sanity_val_steps']=1`

Reply:
this should be ok since the test creates fixed values for the args anyway. 
@Borda is this ok?

Reply:
> P,S : I am new to writing and modifying tests hence i am not sure how to proceed further.

I think what @williamFalcon meant was to make fast_dev_run also run on the test set. Then it would be a kind of unit test because it will check to run on train, val, and test data.